### ISSUE TITLE
Fix build

### ISSUE THREAD

**Issue by brmzkw** [scaleway/image-ubuntu#86]
I tried to fix the image build which is failing for a while now (~3 months), preventing to fix other issues and to release new Ubuntu images.

I'm not sure whether my last commits (c3b73366f 1fb7c97a3c28 4bcbf8d801b5) are the right way to go or not.

Running `make build ARCH=armv7l` from `image-ubuntu/16.04` works on a x86 machine, but running `make build` from the same directory on a C1 server fails.

In short, there's still some work to do to fix the build. To be continued..
